@types/commercetools-uikit__async-select-input/index.d.ts

```diff
@@ -18,7 +18,7 @@ declare module '@commercetools-uikit/async-select-input' {
     menuPortalZIndex: number;
     containerId?: string;
     name: string;
-    value: string | string[] | TAsyncOption;
+    value?: string | string[] | TAsyncOption | null;
```

Comment:
`null` is the fallback value when consumer is able to convert a nullable `data` field into an option.
I prefer this over `undefined`. `null` derives directly from the Apollo result